gedi/gediFinder.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""gediFinder.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1PmKDBLkBxVhf1nrORDspAnqKCb7fkeMt
+
+# GEDI Finder script
+
+This script is meant to create a GediFinder URL and get the corresponding list of granules within a user-defined bounding box. This list can then be used in the Earthdata Search Tool to pull data from within a bounding box.
+"""
+
+# Import necessary packages
+
+# Use requests package to retreive list of files from GEDI Finder URL
+import requests
+
+"""## Define bounding box and other variables
+
+### Bounding box variables
+"""
+
+# Pacific Northwest bbox
+ul_lat = 44.75
+lr_lat = 44.25
+ul_lon = -122.25
+lr_lon = -122.75
+
+"""### Constant values"""
+
+# Server url for LP DAAC which stores GEDI data
+lpdaac = 'https://lpdaacsvc.cr.usgs.gov/services/gedifinder'
+
+# Different levels of GEDI data currently available to the public
+productLevel1B = 'GEDI01_B'
+productLevel2A = 'GEDI02_A'
+productLevel2B = 'GEDI02_B'
+
+# Image verison number
+version = '001'
+
+# Create bounding box string for url
+bbox = ','.join(map(str, [ul_lat, ul_lon, lr_lat, lr_lon]))
+
+# Define output type of url call
+output = 'json'
+
+"""## Join elements of GediFinder URL"""
+
+# Join together components of url
+urlList = [
+       'product=%s' % (productLevel1B),
+       'version=%s' % (version),
+       'bbox=[%s]' % (bbox),
+       'output=%s' % (output)
```

Comment:
Suggestion: these might also be made a bit more readable with the use of f-strings, something like:
```suggestion
       f'product={productLevel1B}',
       f'version={version}',
       f'bbox={bbox}',
       f'output={output}'
```

Reply:
I've never used f-strings before. This looks much better!